borgmatic/config/generate.py

```diff
@@ -24,7 +24,7 @@ def _schema_to_sample_configuration(schema, level=0):
     for each section based on the schema "desc" description.
     '''
     example = schema.get('example')
-    if example:
+    if example is not None:
```

Comment:
This change makes sense to me, but it would be nice to have some test coverage it.